libs/ml/include/ml/dataloaders/tensor_dataloader.hpp

```diff
@@ -336,14 +342,16 @@ struct MapSerializer<fetch::ml::dataloaders::TensorDataLoader<LabelType, InputTy
   static uint8_t const ONE_SAMPLE_LABEL_SHAPE = 15;
   static uint8_t const ONE_SAMPLE_DATA_SHAPES = 16;
 
-  static uint8_t const BATCH_LABEL_DIM = 17;
-  static uint8_t const BATCH_DATA_DIM  = 18;
-  static uint8_t const COUNT           = 19;
+  static uint8_t const BATCH_LABEL_DIM  = 17;
+  static uint8_t const BATCH_DATA_DIM   = 18;
+  static uint8_t const TRAIN_COUNT      = 19;
+  static uint8_t const TEST_COUNT       = 20;
+  static uint8_t const VALIDATION_COUNT = 21;
 
   template <typename Constructor>
   static void Serialize(Constructor &map_constructor, Type const &sp)
   {
-    auto map = map_constructor(19);
+    auto map = map_constructor(21);
```

Comment:
is this 21 a VALIDATION_COUNT? I mean 

auto map = map_constructor(VALIDATION_COUNT);

or it is just a simple 21?..

Reply:
I got it, here 21 is just a total quantity of possible parameters, listed above. Then never mind.